algorithms/src/Kokkos_Sort.hpp

```diff
@@ -51,6 +51,14 @@
 
 #include <Kokkos_Core.hpp>
 
+// The order of includes (firstly oneDPL's, than std's) because of:
+// https://stackoverflow.com/questions/67923287/how-to-resolve-no-member-named-task-in-namespace-tbb-error-when-using-oned/67924408#67924408
```

Comment:
It likely doesn't matter that much because we can't control if `<algorithms>` is included before the oneDPL headers in a downstream application anyway.

Reply:
Yeah, I was a bit too fast with that :- )

Reply:
I've put `COMPILE_DEFINITIONS PSTL_USE_PARALLEL_POLICIES=0 _GLIBCXX_USE_TBB_PAR_BACKEND=0` into oneDPL TPL cmake.

Reply:
Let's see how people like that. :slightly_smiling_face: